deploy.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Usage: ./deploy.sh v1
+# Example: ./deploy.sh v1
+
+set -e
```

Comment:
This is not actually used in the deployment, and was meant to be an example of how to make this work locally.